src/app/shared/pages/collateral/collateral-page/collateral-page.component.html

```diff
@@ -102,6 +102,7 @@ <h2>Sikkerhet i bolig</h2>
     </div>
     <div style='margin-top: 2rem;'>
       <label style='font-weight: bold;' class='selectorLabel'>Har kjøper av boligen en ektefelle som skal bo i boligen, uten å eie den? *</label>
+      <form [formGroup]='coHabitantForm'>
```

Comment:
![image](https://user-images.githubusercontent.com/45078921/156400369-389ca858-5085-4db0-8e00-803b2706c701.png)

Bruk ngSubmit til å sende formet i stedet for (onClick) på next knappen

Reply:
Siden knappen er av typen submit vil dette være beste løsning med forms

Reply:
Enig, det var kun basert på sånn du gjorde det i broker, den kaller direkte på funksjon fra knappen der